src/libcommon/log/customlogwstream.h

```diff
@@ -26,19 +26,45 @@ class CustomLogWStream : private std::wstringstream {
         CustomLogWStream &operator=(const CustomLogWStream &) = delete;
 
         std::wstring str() const { return std::basic_stringstream<wchar_t>::str(); }
+
+        // We need to cast to std::wstringstream as operators<<(std::wstringstream, const wchar_t *str /*and const std::wstring
+        // &str*/) are defined outside of the class std::wstringstream and therefore it is not applicable to the current object
+        // because of the private inheritance
         CustomLogWStream &operator<<(const wchar_t *str) {
-            // We need to cast to std::wstringstream as the operator<<(std::wstringstream, const wchar_t *str) is defined outside
-            // of the class std::wstringstream and therefore it is not applicable to the current object because of the private
-            // inheritance
             static_cast<std::wstringstream &>(*this) << str;
             return *this;
         }
+        CustomLogWStream &operator<<(const std::wstring &str) {
+            static_cast<std::wstringstream &>(*this) << str;
+            return *this;
+        }
+
+        CustomLogWStream &operator<<(bool b) {
```

Comment:
Maybe for later: using `std::boolalpha` would print `true` and `false` instead of `1` and `0`.